docs/reference/rollup/apis/rollup-search.asciidoc

```diff
@@ -28,19 +28,25 @@ expect given the original query.
 ==== {api-path-parms-title}
 
 `<index>`::
-  (Required, string) Index, indices or index-pattern to execute a rollup search
-  against.  This can include both rollup and non-rollup indices.
++
+--
+(Required, string)
+Comma-separated list of data streams and indices used to limit
+the request. Wildcard expressions (`*`) are supported.
+
+This target can include both rollup and non-rollup indices.
 
 Rules for the `index` parameter:
 
-- At least one index/index-pattern must be specified. This can be either a
+- At least one data stream, index, or wildcard expression must be specified. This target can include a
 rollup or non-rollup index.  Omitting the index parameter, or using `_all`, is
 not permitted.
 - Multiple non-rollup indices may be specified
 - Only one rollup index may be specified. If more than one are supplied, an
 exception occurs.
-- Index patterns may be used, but if they match more than one rollup index an
+- Wildcard expressions may be used, but if they match more than one rollup index an
```

Comment:
This part might be a little tricky. Data streams can serve only as non-rollup indices so multiple data streams could be specified just as multiple non-rollup indices may be specified. I don't know how important it is to make all of that clear.